packages/ticker/src/TickerPlugin.ts

```diff
@@ -1,5 +1,6 @@
 import { Ticker } from './Ticker';
 import { UPDATE_PRIORITY } from './const';
+import { IApplicationOptions } from '../../app/src/Application';
```

Comment:
No relative importing of packages/types. This makes the structure of the project more brittle. Simply do `export * from './Application'` in the src/index.ts of app and then import this from `@pixi/app`